doc/source/how-to/continuous-integration.rst

```diff
@@ -176,3 +176,354 @@ Workflow examples are provided for checking :ref:`Coding style`,
+
+Dependabot secrets
+~~~~~~~~~~~~~~~~~~
+
+These secrets in most of the cases a replica of the `Actions secrets`_. 
+
++-------------------------------------+---------------------------------------------+----------------------------------------+----------------------------------------------+
+| **SECRET**                          | **Repository access**                       | **Token**                              | **Description**                              |
++=====================================+=============================================+========================================+==============================================+
+| ``BOT_APPLICATION_ID``              | Same as `Actions secrets`_ equivalent                                                                                               |
++-------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------+
+| ``BOT_APPLICATION_PRIVATE_KEY``     | Same as `Actions secrets`_ equivalent                                                                                               |
++-------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------+
+| ``LICENSE_SERVER``                  | Same as `Actions secrets`_ equivalent                                                                                               |
++-------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------+
+| ``MULTIPR_DEPENDABOT``              | Same as `Actions secrets`_ equivalent                                                                                               |
++-------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------+
+| ``PYANSYS_PYPI_PRIVATE_PAT``        | Same as `Actions secrets`_ equivalent                                                                                               |
++-------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------+
+
+
+.. _organization_tokens:
+
+Organization tokens
+-------------------
+
+To facilitate certain tasks such as authentication, ``Ansys`` and ``Ansys-internal`` organizations provide to the developer with certain tokens or personal access tokens (PATs).
+These tokens are confidential and for internal use only.
+Some of these tokens can be used as GitHub secrets, others must be requested to pyansys.core@ansys.com.
+
+
+.. _github_tokens:
+
+GitHub tokens
+~~~~~~~~~~~~~
+These tokens are meant to be used within GitHub to provide access and permissions to different tasks and repositories.
```

Comment:
The first sentence of the paragraph should use the full noun string so as to be clear, not these tokens but GitHub tokens....

Reply:
Got it. Thank you!